sections/futures/ProfitCalculator/PnLs.tsx

```diff
@@ -0,0 +1,59 @@
+import { ethers } from 'ethers';
+import { BigNumber } from '@ethersproject/bignumber';
+
+import StatWithContainer from './StatWithContainer';
+import { PositionSide } from '../types';
+
+export const PnLs = (props: {
+	scalar: number;
+	stopLoss: BigNumber;
+	exitPrice: BigNumber;
+	entryPrice: BigNumber;
+	amountInAsset: BigNumber;
+	leverageSide: PositionSide;
```

Comment:
You can pass properties directly, so we can avoid using `props.*`. Set the types separately and do something like:

```
PnLs = ({
    entryPrice,
    exitPrice,
    ...
}: YourType)
```